diff --git a/node_modules/@sphereon/oid4vci-issuer/dist/tokens/index.js b/node_modules/@sphereon/oid4vci-issuer/dist/tokens/index.js
index b175edd..8fb21e0 100644
--- a/node_modules/@sphereon/oid4vci-issuer/dist/tokens/index.js
+++ b/node_modules/@sphereon/oid4vci-issuer/dist/tokens/index.js
@@ -15,10 +15,10 @@ const uuid_1 = require("uuid");
 const functions_1 = require("../functions");
 const generateAccessToken = (opts) => __awaiter(void 0, void 0, void 0, function* () {
     const { accessTokenIssuer, alg, accessTokenSignerCallback, tokenExpiresIn, preAuthorizedCode } = opts;
-    const iat = new Date().getTime();
+    const iat = Math.floor(new Date().getTime()/1000);
     const jwt = {
-        header: { typ: 'JWT', alg: alg !== null && alg !== void 0 ? alg : oid4vci_common_1.Alg.ES256K },
-        payload: Object.assign({ iat, exp: tokenExpiresIn, iss: accessTokenIssuer }, (preAuthorizedCode && { preAuthorizedCode })),
+        header: { typ: 'JWT', alg: alg !== null && alg !== void 0 ? alg : oid4vci_common_1.Alg.EdDSA },
+        payload: Object.assign({ iat, exp: iat+100, iss: accessTokenIssuer }, (preAuthorizedCode && { preAuthorizedCode })),
     };
     return yield accessTokenSignerCallback(jwt);
 });
